integration-test/src/main/java/com/scalar/db/api/DistributedTransactionIntegrationTestBase.java

```diff
@@ -910,7 +910,7 @@ public void resume_WithBeginningTransaction_ShouldReturnBegunTransaction()
     DistributedTransaction resumed = manager.resume(transaction.getId());
 
     // Assert
-    assertThat(resumed).isEqualTo(transaction);
+    assertThat(resumed.getId()).isEqualTo(transaction.getId());
```

Comment:
Instead of comparing a transaction object itself, we should compare the transaction ID.

Reply:
Looks good. Another option is overriding `DistributedTransaction#equals` to compare transaction ID?

Reply:
I think, yes, that's an option. But I'm not sure it's okay to consider only transaction ID for the equivalence of transactions. So I will keep it for now. Thank you for the comment.